web-common/src/features/dashboards/pivot/pivot-expansion.ts

```diff
@@ -278,6 +278,13 @@ export function queryExpandedRowMeasureValues(
         mergeFilters(filterForRowDimensionAxes, config.whereFilter) ??
         createAndExpression([]);
 
+      // Use rowLimit if set, otherwise default to 100
+      // TODO: Add See more option
+      const expandLimitToApply =
+        config.pivot.rowLimit !== undefined
+          ? config.pivot.rowLimit.toString()
+          : "100";
```

Comment:
nit: this limit should stay along with `PIVOT_ROW_LIMIT_OPTIONS`.

Also the hardcoded limit to `100` is already in `PIVOT_ROW_LIMIT_OPTIONS`. Since all and `100` are the same we should get rid of `100` option.

Reply:
Added a See more button based on product's recommendation. The max limit is still capped to 100.